clarity/src/vm/representations.rs

```diff
@@ -455,12 +455,16 @@ pub struct SymbolicExpression {
     pub id: u64,
 
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub span: Span,
 
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub pre_comments: Vec<(String, Span)>,
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub end_line_comment: Option<String>,
+    #[serde(default)]
     #[cfg(feature = "developer-mode")]
```

Comment:
Nit: can you swap these two lines so that they're all in the same order?

Reply:
Oh right, fixed